javascript/javascript1/week2/candyhelper.js

```diff
@@ -0,0 +1,46 @@
+const boughtCandyPrices=[];
+
+const candyType=['sweet', 'chocolate', 'toffe', 'chewing-gum'];
+const pricePergram=[0.5, 0.7, 1.1,0.03];
+
+function addCandy(candyType, weight){
+    if(candyType==='sweet'){
```

Comment:
Check switch and how it can be used here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
do I need to use switch and case here, instead of a function and else if statement ?

Reply:
That is just a suggestion to try to use switch.